azdev/utilities/command.py

```diff
@@ -57,11 +61,13 @@ def cmd(command, message=False, show_stderr=True, raise_error=False, **kwargs):
         display(message)
 
     logger.info("Running: %s", command)
+    cmd_args = shlex.split(command)
+    if cmd_args[0] == "az" and IS_WINDOWS:
+        cmd_args[0] = "az.bat"
     try:
         output = subprocess.check_output(
```

Comment:
`subprocess.check_output` can be replaced by `run(..., check=True, stdout=PIPE).stdout`: https://docs.python.org/3/library/subprocess.html#subprocess.check_output